moderators.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Moderators"
+---
+
+A [moderator](governance.md#moderator) is a volunteer gatekeeper
+who manages contributions and discussions from [contributors](governance.md#contributor) and [users](governance.md#user). 
+[Moderators](governance.md#moderator) are vital to the health and success of R-multiverse.
+This page explains the prerequisites, selection criteria, and scope
+of the [moderator](governance.md#moderator) role.
+
+# Prerequisites
+
+[Moderators](governance.md#moderator) must be:
+
+1. Proficient in [R package development](https://r-pkgs.org/).
+1. Comfortable managing content on [GitHub](https://github.com) such as repositories, pull requests, issues, and discussions.
+1. Prepared to facilitate the [process to contribute packages to R-multiverse](https://r-multiverse.org/contributors.html).
+1. Prepared to facilitate discussions and answer questions on the [`r-multiverse/help`](https://github.com/r-multiverse/help)
+[issue](https://github.com/r-multiverse/help/issues) and [discussion](https://github.com/r-multiverse/help/discussions) forums.
+1. Prepared to enforce [all R-multiverse policies](policies.md).
+
+# Selection
+
+[Administrators](governance.md#administrator) are responsible for selecting [moderators](governance.md#moderator).
+Beyond the [prerequisites](#prerequisites) above,
+[administrators](governance.md#administrator) choose [moderators](governance.md#moderator) based on trust.
+A [moderator](governance.md#moderator) should have a positive, strong, visible, verifiable presence in the R community.
+If you would like to be a [moderator](governance.md#moderator),
+please help [R-multiverse stakeholders](governance.md#r-multiverse-stakeholders) understand who you are and what you work on.
+There are many ways to grow your public reputation, including but not limited to:
+
+1. Contributing packages to well-known repositories (or R-multiverse).
+1. [Reviewing packages](https://ropensci.org/software-review/) for [rOpenSci](https://ropensci.org).
+1. Speaking at an [R-focused conference](https://www.r-project.org/conferences/).
+1. Contributing to an [R Consortium ISC working group](https://r-consortium.org/all-projects/isc-working-groups.html).
+1. Contributing to an [ASA](https://community.amstat.org/) group such as [openstatsware](https://www.openstatsware.org).
```

Comment:
maybe add a last item with ellipsis?

what about community service such as curating R Weekly, organizing conferences/RUGs/R-Ladies chapters, etc. I feel it could be an explicit item.

Reply:
Those are good examples. 00f5ec469133fba66b7a3858eea665c00b0684a0 adds them and emphasizes that each moderator has their own type of impact.